README.md

```diff
@@ -70,6 +70,8 @@ Use the [installation instructions in the Hugo documentation](http://gohugo.io/o
 
     Go expects all of your libraries to be found in`$GOPATH`.
 
+7. If you are changing files in the `commands` directory, you will want to make a local change to `main.go` so that it imports from `./commands` instead of github.
```

Comment:
This isn't need - and may lead to faulty commits. 

The trick I use is to just clone spf13's Hugo (or go get) into `$GOPATH/src/github.com`, then add my Hugo fork as a remote. The imports are unchanged and everything just works as expected.